packit_service/worker/testing_farm.py

```diff
@@ -237,6 +242,10 @@ def distro2compose(self, distro: str) -> str:
                 return "RHEL-7-LatestReleased"
             if compose == "RHEL-8":
                 return "RHEL-8.5.0-Nightly"
+            if compose == "Oracle-Linux-7":
+                return "Oracle-Linux-7.9"
+            if compose == "Oracle-Linux-8":
+                return "Oracle-Linux-8.3"
```

Comment:
```suggestion
                return "Oracle-Linux-8.4"
```
We need OL 8.4 as [we don't allow converting OL 8.3 anymore](https://github.com/oamg/convert2rhel/pull/299/files#diff-1e030442df599479d1af356a7eaa61a744973513548846a1a885b8eaca6f489cR38).